README.md

```diff
@@ -25,26 +25,28 @@ MapKnitter is broken into three major components:
 
 Another moving part is the new-ish Annotations 2.0 which uses Leaflet.Illustrate (https://github.com/manleyjster/Leaflet.Illustrate) to provide rich annotation on top of maps. 
 
-## License
+## Quick install
 
-Map Knitter is free software: you can redistribute it and/or modify
-it under the terms of the GNU General Public License as published by
-the Free Software Foundation, either version 3 of the License, or
-(at your option) any later version.
+We provide an install script for Amazon's Cloud9 service, which provides standard Linux virtual machines with a web-based editor: https://aws.amazon.com/cloud9/
 
-Map Knitter is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU General Public License for more details.
+To use it:
 
-You should have received a copy of the GNU General Public License
-along with Map Knitter.  If not, see <http://www.gnu.org/licenses/>.
+1. Click `Create new workspace`
+2. Name your workspace and enter `https://github.com/USERNAME/mapknitter` (where `USERNAME` is your GitHub username) under `Clone from Git or Mercurial URL`
+3. Choose `Ruby` template
+4. Click `Create workspace`
+5. Once it loads, in the `bash` console at the bottom of the screen, type `chmod 777 install_cloudnine.sh && ./install_cloudnine.sh`
```

Comment:
Why not `chmod` the script already and commit it? :thinking:

Reply:
Really good idea @alaxalves! I've added the permissions to git.
Thanks